lib/ts/superTokens.tsx

```diff
@@ -195,10 +196,13 @@ export default class SuperTokens {
             queryParams.redirectToPath = getCurrentNormalisedUrlPath().getAsStringDangerous();
         }
 
-        let redirectUrl = await this.getRedirectUrl({
-            action: "TO_AUTH",
-            showSignIn: options.show === "signin",
-        });
+        let redirectUrl = await this.getRedirectUrl(
+            {
+                action: "TO_AUTH",
+                showSignIn: options.show === "signin",
+            },
+            getNormalisedUserContext(undefined)
```

Comment:
instead of passing in undefined, you should add a userContext input to the `redirectToAuth` function. And of course, the user facing version of this function (in the index.ts) will have the userContext as optional

Reply:
done